### PR TITLE
Minor updates to README

### PR DESCRIPTION
**Pull request by docelic**
(This PR is dependent on #484 because it mentions running just 'make' as a way to build including downloading the shards, which #484 makes possible.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,20 +58,22 @@ Ensure you have the necessary dependencies:
...
-Once you have these dependencies, You can build the `amber` tool from source:
+Once you have these dependencies, you can build `amber` from source:
 
 
 ```shellsession
 $ git clone git@github.com:amberframework/amber.git
-$ cd amber/
-$ shards install
+$ cd amber
+$ make
```
Comment (eliasjpr) [inline README.md]: you have to run the shards install before running make

File: README.md [hunk 2]
```diff
@@ -213,7 +215,7 @@ It's not enough to be brilliant when you're alone in your programming lair. You
 
 We have adopted the Contributor Covenant to be our [CODE OF CONDUCT](.github/CODE_OF_CONDUCT.md) guidelines for Amber.
 
-## Have a Amber based project?
+## Have an Amber-based Project?
```
Comment (eliasjpr) [inline README.md]: 👍

File: README.md (hunk 1 above)
Comment (eliasjpr) [reply README.md]: It seems that this is related to this change https://github.com/amberframework/amber/pull/484/files#diff-b67911656ef5d18c4ae36cb6741b7965R14

If so running `make install` is sufficient here?

Comment (docelic) [reply README.md]: Yes, 'make install' would download deps, compile, and install. (Assuming that the privileges under which process is running are enough to do an installation in /usr/local/bin)

**Pull request merged**